Intro/README.md

```diff
@@ -0,0 +1,455 @@
+# Introduction to TPMs
+
+Trusted Platform Modules (TPMs) are a large and complex topic, made all
+the more difficult to explain by the intricate relationships between the
+relevant concepts.  This is an attempt at a simple explanation --
+simpler than reading hundreds of pages of documents, but then too, too
+light on detail to be immediately useful.
+
```

Comment:
There's stuff here that you are free to take and incorporate : https://github.com/nokia/TPMCourse/tree/master/docs
Check the license though :-)

Reply:
Will do.

Reply:
Linking to it should be fine!

Reply:
@nicowilliams I also think linking to external sources is totally fine. 

Nokia's TPM Course is a great resource. Cheers @iolivergithub